notebooks/feature_engineering/raw/ex1.ipynb

```diff
@@ -271,7 +257,7 @@
     "valid = clicks_srt[-valid_rows * 2:-valid_rows]\n",
     "test = clicks_srt[-valid_rows:]\n",
     "\n",
-    "q_5.check()"
+    "q_5.assert_check_passed()"
```

Comment:
@mcleonard Using these asserts will make sure we are alerted if something changes (either underlying library versions, data, etc) that breaks checking code.

There is also an assert_check_failed() method. So we can add a lot of tests and be confident about what does or doesn't pass our checking code.